tox.ini

```diff
@@ -31,7 +31,6 @@ commands =
     astropydev: bash -ec "for f in ../examples/*/*.py; do [[ $f == *skip* ]] && continue; python "$f"; done"
 
 [testenv:build_docs]
-basepython = python3.7
```

Comment:
Y dis?

Reply:
I don't have 3.7 installed locally so I couldn't build the docs. Is there a good reason to keep it?

Reply:
0.

If the docs varied on python version, I would be very worried.